From 37966294de427e19bf19ceea202fe15545ab345c Mon Sep 17 00:00:00 2001
From: Alan Winkowski <walan@marvell.com>
Date: Tue, 30 Jan 2018 19:18:03 +0200
Subject: [PATCH 2209/2241] ppv2: export functions used in mv-sysfs

- export mv_gop110_mib_counters_stat_update() and
  mv_pp2x_counters_stat_update() functions, as they
  are used in Marvell's sysfs

Change-Id: Ib650cf53bfc5da305e80a4cc312faf31e8297d79
Signed-off-by: Alan Winkowski <walan@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/49818
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-by: Yuval Caduri <cyuval@marvell.com>
---
 drivers/net/ethernet/marvell/mvpp2x/mv_gop110_hw.c | 1 +
 drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_hw.c   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/drivers/net/ethernet/marvell/mvpp2x/mv_gop110_hw.c b/drivers/net/ethernet/marvell/mvpp2x/mv_gop110_hw.c
index 9d0aa4c..c9047f4 100644
--- a/drivers/net/ethernet/marvell/mvpp2x/mv_gop110_hw.c
+++ b/drivers/net/ethernet/marvell/mvpp2x/mv_gop110_hw.c
@@ -2665,6 +2665,7 @@ void mv_gop110_mib_counters_stat_update(struct gop_hw *gop, int port, struct gop
 	gop_statistics->late_collision += mv_gop110_mib_read64(gop, port,
 							MV_MIB_LATE_COLLISION);
 }
+EXPORT_SYMBOL(mv_gop110_mib_counters_stat_update);
 
 void mv_gop110_mib_counters_clear(struct gop_hw *gop, int port)
 {
diff --git a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_hw.c b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_hw.c
index 4628728..30ad2fd 100644
--- a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_hw.c
+++ b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_hw.c
@@ -6572,6 +6572,7 @@ void mv_pp2x_counters_stat_update(struct mv_pp2x_port *port,
 	}
 	preempt_enable();
 }
+EXPORT_SYMBOL(mv_pp2x_counters_stat_update);
 
 /*  Clear Mvpp2x counter statistic */
 void mv_pp2x_counters_stat_clear(struct mv_pp2x_port *port)
-- 
2.7.4

